<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lbl_lastname">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Lastname</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_lastname">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_register">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>260</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Register</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_firstname">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>60</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_firstname">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>62</y>
      <width>49</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Firstname</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spx_level">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>100</y>
      <width>42</width>
      <height>25</height>
     </rect>
    </property>
    <property name="maximum">
     <number>10</number>
    </property>
    <property name="value">
     <number>5</number>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_level">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>103</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Level</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="dt_birthdate">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>140</y>
      <width>110</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_birthdate">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Birthdate</string>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>190</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QComboBox" name="cb_lang">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>220</y>
      <width>72</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_lang">
    <property name="geometry">
     <rect>
      <x>31</x>
      <y>224</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Langue</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_joke">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>260</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Joke</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_add_langs">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>260</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Langs</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_clear_langs">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>260</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear Langs</string>
    </property>
   </widget>
   <widget class="QListView" name="lv_alllangs">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>50</y>
      <width>91</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
